### PR TITLE
create the machine arm for haas example #1672

### PR DESCRIPTION
**Pull request by wushengtws**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : dc2eb30baee7c29eebb73ed18cb76504f9f3a269

**Review by ethanlcz — APPROVED** [review]

File: haas_lib_bundles/python/docs/examples/machine_arm/esp32/README.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+# 三元组信息
+productKey = "产品密钥" #需要填入物联网云平台申请到的productKey信息
+deviceName = "设备名称" #需要填入物联网云平台申请到的deviceName信息
+deviceSecret = "设备密钥" #需要填入物联网云平台申请到的deviceSecret信息
+
+# 物联网设备实例
+device = None
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/machine_arm/esp32/README.md:203]: no security risk

File: haas_lib_bundles/python/docs/examples/machine_arm/esp32/code/main.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+# 三元组信息
+productKey = "产品密钥" #需要填入物联网云平台申请到的productKey信息
+deviceName = "设备名称" #需要填入物联网云平台申请到的deviceName信息
+deviceSecret = "设备密钥" #需要填入物联网云平台申请到的deviceSecret信息
+
+# 物联网设备实例
+device = None
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/machine_arm/esp32/code/main.py:27]: no security risk

File: haas_lib_bundles/python/docs/examples/machine_arm/esp32/README.md [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+import pca9685
+
+robotDev = None
+
+#########
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/machine_arm/esp32/README.md:195]: no security risk

File: haas_lib_bundles/python/docs/examples/machine_arm/esp32/code/main.py [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+import pca9685
+
+robotDev = None
+
+#########
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/machine_arm/esp32/code/main.py:19]: no security risk

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**